pkgs/applications/video/mpv/default.nix

```diff
@@ -178,68 +193,181 @@ in stdenv'.mkDerivation (finalAttrs: {
-    ++ lib.optionals (stdenv.isDarwin && swiftSupport) [ AVFoundation CoreMedia ];
+  ]
+  ++ lib.optionals alsaSupport [
+    alsa-lib
+  ]
+  ++ lib.optionals archiveSupport [
+    libarchive
+  ]
+  ++ lib.optionals bluraySupport [
+    libbluray
+  ]
+  ++ lib.optionals bs2bSupport [
+    libbs2b
+  ]
+  ++ lib.optionals cacaSupport [
+    libcaca
+  ]
+  ++ lib.optionals cddaSupport [
+    libcdio
+    libcdio-paranoia
+  ]
+  ++ lib.optionals cmsSupport [
+    lcms2
+  ]
+  ++ lib.optionals drmSupport [
+    libdrm
+    mesa
+  ]
+  ++ lib.optionals dvdnavSupport [
+    libdvdnav
+    libdvdnav.libdvdread
+  ]
+  ++ lib.optionals jackaudioSupport [
+    libjack2
+  ]
+  ++ lib.optionals javascriptSupport [
+    mujs
+  ]
+  ++ lib.optionals libpngSupport [
+    libpng
+  ]
+  ++ lib.optionals openalSupport [
+    openalSoft
+  ]
+  ++ lib.optionals pipewireSupport [
+    pipewire
+  ]
+  ++ lib.optionals pulseSupport [
+    libpulseaudio
+  ]
+  ++ lib.optionals rubberbandSupport [
+    rubberband
+  ]
+  ++ lib.optionals screenSaverSupport [
+    libXScrnSaver
+  ]
+  ++ lib.optionals sdl2Support [
+    SDL2
+  ]
+  ++ lib.optionals sixelSupport [
+    libsixel
+  ]
+  ++ lib.optionals speexSupport [
+    speex
+  ]
+  ++ lib.optionals theoraSupport [
+    libtheora
+  ]
+  ++ lib.optionals vaapiSupport [
+    libva
+  ]
+  ++ lib.optionals vapoursynthSupport [
+    vapoursynth
+  ]
+  ++ lib.optionals vdpauSupport [
+    libvdpau
+  ]
+  ++ lib.optionals vulkanSupport [
+    shaderc
+    vulkan-headers
+    vulkan-loader
+  ]
+  ++ lib.optionals waylandSupport [
+    libxkbcommon
+    wayland
+    wayland-protocols
+  ]
+  ++ lib.optionals x11Support [
+    libX11
+    libXext
+    libGLU
+    libGL
+    libXxf86vm
+    libXrandr
+    libXpresent
+  ]
+  ++ lib.optionals xineramaSupport [
+    libXinerama
+  ]
+  ++ lib.optionals xvSupport [
+    libXv
+  ]
+  ++ lib.optionals zimgSupport [
+    zimg
+  ]
+  ++ lib.optionals stdenv.isLinux [
+    nv-codec-headers
+  ]
+  ++ lib.optionals stdenv.isDarwin [
+    libiconv
+  ]
+  ++ lib.optionals stdenv.isDarwin [
+    Accelerate
+    CoreFoundation
+    Cocoa
+    CoreAudio
+    MediaPlayer
+    VideoToolbox
+  ]
+  ++ lib.optionals (stdenv.isDarwin && swiftSupport) [
+    AVFoundation
+    CoreMedia
+  ];
+
+  mesonAutoFeatures = "auto";
+
+  postPatch = lib.concatStringsSep "\n" [
+    # Don't reference compile time dependencies or create a build outputs cycle
+    # between out and dev
+    ''
+    substituteInPlace meson.build \
+      --replace-fail "conf_data.set_quoted('CONFIGURATION', configuration)" \
+                     "conf_data.set_quoted('CONFIGURATION', '<ommited>')"
+    ''
+    # A trick to patchShebang everything except mpv_identify.sh
+    ''pushd TOOLS''
+    ''mv mpv_identify.sh mpv_identify''
+    ''patchShebangs *.py *.sh''
+    ''mv mpv_identify mpv_identify.sh''
+    ''popd''
```

Comment:
This should become a multi line string again

Reply:
I have mixed feelings about this. The indentation of multiline strings is less than good.